### PR TITLE
feat(project-setup): web app project setup finished

### PR DESCRIPTION
**Pull request by glebhg**
**What**

- ESlint, Stylellint, Prettier configuration added
- Jest is added with a dummy test
- Git hooks added
- README added
- Simple Material setup added
- Simple HTML added

closes to #4

### CODE REVIEW

**Review by NatalliaMil — DISMISSED** [review]

File: web/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,9 @@
+root = true
```
Comment (glebhg) [inline web/.editorconfig:1]: https://editorconfig.org/

File: web/babel.config.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
```
Comment (glebhg) [inline web/babel.config.js:1]: https://babeljs.io/

File: web/.stylelintrc.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (glebhg) [inline web/.stylelintrc.json:1]: https://stylelint.io/

File: web/.prettierrc.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
Comment (glebhg) [inline web/.prettierrc.js:1]: https://prettier.io/

File: web/.eslintrc.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+module.exports = {
```
Comment (glebhg) [inline web/.eslintrc.js:1]: https://eslint.org/

**Pull request merged**